README.md

```diff
@@ -1,6 +1,10 @@
-# AutoFlashGUI
```

Comment:
Revert README.md to its upstream version, then include new models where appropriate